Xamarin.Forms.Platform.Android/AppCompat/TabbedPageRenderer.cs

```diff
@@ -263,6 +264,34 @@ void OnChildrenCollectionChanged(object sender, NotifyCollectionChangedEventArgs
 			UpdateIgnoreContainerAreas();
 		}
 
+		void TeardownPage(Page page, int i)
+		{
+			page.PropertyChanged -= OnPagePropertyChanged;
+		}
+
+		void SetupPage(Page page, int i)
+		{
+			page.PropertyChanged += OnPagePropertyChanged;
+		}
+
+		void Reset()
+		{
+			var i = 0;
+			foreach (var page in Element.Children)
+				SetupPage(page, i++);
+		}
+
+		private void OnPagePropertyChanged(object sender, PropertyChangedEventArgs e)
```

Comment:
Our [coding style](https://github.com/xamarin/Xamarin.Forms#coding-style) excludes the explicit `private` key word.